wordpress-mobile/WordPress-iOS
# Migrate ThemeServiceRemote into WordPressKit

Fixes #7333 

To test:
1. Clean, Build & Test main WordPress target.
2. Run the app.
3. Log in to a WordPress.com account.
4. Use the Theme feature on an existing .com site and make sure it works as expected.

Needs review: @bummytime
